package.json

```diff
@@ -7,7 +7,13 @@
     "eslint:ci": "eslint . --report-unused-disable-directives --ext .js,.ts,.tsx --max-warnings 0",
     "prettier": "pretty-quick --ignore-path .eslintignore",
     "prettier:all": "prettier --write . --ignore-path .eslintignore",
-    "update-netlify-ignore": "node ./update-netlify-ignore.js code-infra-dashboard"
+    "update-netlify-ignore": "node ./update-netlify-ignore.js code-infra-dashboard",
+    "release:prepare": "pnpm install && pnpm release:build",
+    "release:version": "lerna version --no-changelog --no-push --no-git-tag-version --no-private",
+    "release:build": "pnpm --filter \"./packages/**\" run build",
+    "release:publish": "pnpm publish --recursive --tag latest",
+    "pkg-pr-new-packages": "pnpm ls -r --parseable --depth -1 -F \"./packages/**\"",
+    "pkg-pr-new-release": "pnpm dlx pkg-pr-new publish $(pnpm -s pkg-pr-new-packages) --pnpm --comment=off --peerDeps --compact"
```

Comment:
I wonder if there's a point in having this defined as an npm script. `pnpm dlx pkg-pr-new publish` can only be run from a GH action, so it can be defined there (and not clutter the package.json unnecessarily)

Reply:
We currently tend to treat the workspace root package.json as a sort of poor man's makefile. I'm personally quite used to that approach and have no big problem with a large scripts section. But happy to move this out.